### ISSUE TITLE
Choose template

### ISSUE THREAD

**Issue by tsalo** [NBCLab/NBCLab.github.io#1]
A few great examples of lab websites we can use (first link is site, second is GitHub repo):
- Bedford Lab
    - http://bedford.io
    - https://github.com/blab/blotter)
- Drummond Lab
    - http://drummondlab.org
    - https://github.com/drummondlab/drummondlab.github.io
- COMBINE Lab
    - http://combine-lab.github.io
    - https://github.com/COMBINE-lab/COMBINE-lab.github.io
- Whitaker Lab
    - https://whitakerlab.github.io
    - https://github.com/WhitakerLab/whitakerlab.github.io
- Caporaso Lab
    - http://caporasolab.us
    - https://github.com/caporaso-lab/caporaso-lab.github.io

**Comment by tsalo** [comment]
I think the Bedford lab site would be a great template. We could switch the blog out for a news stream. The only potential difficulty is the projects. That lab seems to work primarily in public repositories, so there are integrations in the project pages that wouldn't fit with how we do things.

**Comment by angielaird** [comment]
Agreed, I like the Bedford site the best. As ATHENA matures, more of our work will be disseminated through public repos, so we can develop the template now and know that it will evolve into something that works well.
